Experimental/DistributedAuthoritySample/Assets/Scripts/Player/AvatarTransform.cs

```diff
@@ -1,17 +1,16 @@
 using Unity.Netcode.Components;
 using UnityEngine;
-using UnityEngine.InputSystem;
-using com.unity.multiplayer.samples.distributed_authority.input;
+using Unity.Multiplayer.Samples.SocialHub.Input;
 
-namespace com.unity.multiplayer.samples.distributed_authority.gameplay
+namespace Unity.Multiplayer.Samples.SocialHub.Player
 {
     [RequireComponent(typeof(Rigidbody))]
     public class AvatarTransform : NetworkTransform
     {
         [SerializeField]
         Rigidbody m_Rigidbody;
         [SerializeField]
-        PlayerInput m_PlayerInput;
+        MonoBehaviour m_PlayerInput;
```

Comment:
is this intended? MonoBehaviour could be anything

Reply:
I initially didn't want to include the InputSystem assembly inside the Player's, but yes let's bind them.